cmd/apiserver_receive_adapter/main.go

```diff
@@ -59,14 +59,16 @@ func (s *StringList) Decode(value string) error {
 }
 
 type envConfig struct {
-	Namespace     string     `envconfig:"SYSTEM_NAMESPACE" default:"default"`
-	Mode          string     `envconfig:"MODE"`
-	SinkURI       string     `split_words:"true" required:"true"`
-	ApiVersion    StringList `split_words:"true" required:"true"`
-	Kind          StringList `required:"true"`
-	Controller    []bool     `required:"true"`
-	LabelSelector StringList `envconfig:"SELECTOR" required:"true"`
-	Name          string     `envconfig:"NAME" required:"true"`
+	Namespace       string     `envconfig:"SYSTEM_NAMESPACE" default:"default"`
+	Mode            string     `envconfig:"MODE"`
+	SinkURI         string     `split_words:"true" required:"true"`
+	ApiVersion      StringList `split_words:"true" required:"true"`
+	Kind            StringList `required:"true"`
+	Controller      []bool     `required:"true"`
+	LabelSelector   StringList `envconfig:"SELECTOR" required:"true"`
+	OwnerApiVersion StringList `envconfig:"OWNER_API_VERSION" required:"true"`
```

Comment:
we require the new setting?

Reply:
This is internally required. The receive adapter is created by the API source controller, not the end-user.